scripts/pricefeeds.py

```diff
@@ -238,18 +240,25 @@ def get_btsprice():
     for idx in range(0, len(price[base][core_symbol])) :
      price[quote][core_symbol].append( (float(price[base][core_symbol][idx] /ratio)))
      volume[quote][core_symbol].append((float(volume[base][core_symbol][idx]/ratio)))
+     #print("price["+quote+"]["+core_symbol+"]=%s"%(price[quote][core_symbol]))
+     #print("volume["+quote+"]["+core_symbol+"]=%s"%volume[quote][core_symbol])
+
+ #print("*****************************************************************************************")
 
  for base in _bases :
   for quote in asset_list_publish :
    for ratio in price[ base ][ quote ] :
     for idx in range(0, len(price[base][core_symbol])) :
-     price[core_symbol][quote].append( (float(price[base][core_symbol][idx] /ratio)))
-     volume[core_symbol][quote].append((float(volume[base][core_symbol][idx]/ratio)))
+     price[quote][core_symbol].append( (float(price[base][core_symbol][idx] /ratio)))
```

Comment:
My philosophy was to have the base first and than chose the quote .. but I think you can do so too .. just need to think again about the prices pulled from the external exchanges ..